### ISSUE TITLE
Create and use a JSON schema for workflows

### ISSUE THREAD

**Issue by JustinTervala** [iadgov/WALKOFF#169]
A JSON Schema for workflows should be created and workflows should be verified against it inside `core.jsonplaybookloader`. Additionally, it would be good documentation to put on the `gh-pages` branch.

**Comment by JustinTervala** [comment]
This issue will be irrelevant once the development branch is merged into master. In that branch, we have moved the workflows and their execution elements into a database
